pkg/cmd/server/server.go

```diff
@@ -88,6 +88,7 @@ const (
 
 	defaultBackupSyncPeriod           = time.Minute
 	defaultStoreValidationFrequency   = time.Minute
+	defaultGarbageCollectionFrequency = time.Hour
```

Comment:
Would it make sense to only have one defined default?

Here's another locatiion.
https://github.com/vmware-tanzu/velero/blob/main/pkg/controller/gc_controller.go#L41